patches/minecraft/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -716,7 +716,7 @@
 +
 +   @Override
 +   public <T> net.minecraftforge.common.util.LazyOptional<T> getCapability(net.minecraftforge.common.capabilities.Capability<T> capability, @Nullable Direction facing) {
-+      if (this.m_6084_() && capability == net.minecraftforge.common.capabilities.ForgeCapabilities.ITEM_HANDLER) {
++      if (capability == net.minecraftforge.common.capabilities.ForgeCapabilities.ITEM_HANDLER && this.m_6084_()) {
```

Comment:
This is probably the most important location to fix - would it be worth flipping the order of the checks in other implementations of `getCapability`?

Reply:
Didn't realize it was done in more than one place. Will update.